stages/org.osbuild.greenboot

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+"""
+Configure greenboot
+
+Update configuration of greenboot in /etc/greenboot/greenbot.conf.
+"""
+
+import fileinput
+import sys
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["config"],
+"properties": {
+  "config": {
+    "additionalProperties": false,
+    "description": "greenboot config options",
+    "type": "object",
+    "properties": {
+      "monitor_services": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+      }
+    }
+  }
+}
+"""
+
+
+def main(tree, options):
+    greenboot_conf = options.get("config", {})
+    changes = greenboot_conf.copy()
+    config_file = f"{tree}/etc/greenboot/greenboot.conf"
+
+    # For each of the configured options, find the first non-commented out instance
+    # of the option and replace it (if necessary). If it does not already exist, append
+    # the option to the end of the file.
+    # Keys are case case sensitive.
+    with fileinput.input(files=(config_file), inplace=True) as f:
+        for line in f:
+            line_list = line.split('=')
+            if len(line_list) == 2:
+                key, current_value = line_list
+                value = changes.pop(key, None)
+                if value is not None:
+                    if key == "monitor_services":
+                        svcs = current_value.split(" ")
```

Comment:
According to the schema the value in the options dict is already an array of strings, no?

Reply:
yeah, my bad here

Reply:
so the schema refers to `value` not `current_value` - `current_value` is the string from the config file so it still needs to be exploded with split

Reply:
Sorry, yeah, got confused with all the `values`. If you do another pass maybe just drop the `_value`, adds little but apparently confuses me ;)

Reply:
done, it's now called just `current` which I think it makes sense